Tickets: Added remote tickets (#653)
EuroPython/website: src/content/pages/tickets.mdx

EuroPython 2024 will be held on 8-14 July. In person, it will be held at
the [Prague Congress Centre (PCC)](https://www.praguecc.cz/en/homepage), the Czech Republic.
For those who cannot physically join us but still want to support the community,

> we have the remote ticket option. If you do not have a remote ticket, the livestreaming will

Reviewer: ```suggestion
we have the remote ticket option. Without a remote ticket, the recordings will
```

We understand attendees have different needs, so we've created three types of
ticket to give folks maximum flexibility and value for money:

- **Conference Tickets** (in-person & remote):
    - **In-person**: Access to Conference & Sprint Weekend (10-14 July).
      There will be limited access to specific sponsored/special workshops during
      the Workshop/Tutorial Days (8-9 July).

>     - **Remote**: remote participation during the Conference Days (10-12 July):

Reviewer 2: ```suggestion
    - **Remote**: Remote participation during the Conference Days (10-12 July):
```

> - **Conference Tickets** (in-person & remote):
>     - **In-person**: Access to Conference & Sprint Weekend (10-14 July).
>       There will be limited access to specific sponsored/special workshops during
>       the Workshop/Tutorial Days (8-9 July).
>     - **Remote**: remote participation during the Conference Days (10-12 July):
>       watch the live talks, keynotes & panels in all 6 tracks, engage in live text-based Q&A,
>       and interact with speakers and in-person attendees in chat channels.

Reviewer 2: ```suggestion
- **Conference Tickets** (in-person & remote):
    - **In-person**: Access to Conference & Sprint Weekend (10-14 July)
      - Limited access to specific sponsored/special workshops
      during the Workshop/Tutorial Days (8-9 July) are also included.
    - **Remote**: Remote participation during the Conference Days (10-12 July)
      - Watch the live talks, keynotes & panels in all 6 tracks,
      - Engage in live text-based Q&A,
      - And interact with speakers and in-person attendees in chat channels.
```

IMHO a structure like this is more readable
